src/stores/slices/items/itemsSlice.js

```diff
@@ -0,0 +1,61 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { toast } from 'react-toastify';
+import * as yup from 'yup';
+import {
+  getItems,
+  updateItem as clientUpdateItem,
+  deleteItem as clientDeleteItem,
+  addItem as clientAddItem,
+} from '@api/items';
+import { checkFileRequired, checkFileSize } from '@utils';
+
+export const itemSchema = yup
+  .object({
+    name: yup.string().min(3).max(40).required().trim(),
+    price: yup.number().positive().integer().required(),
+    image: yup
+      .mixed()
+      .required()
+      .test('required', 'File is required.', checkFileRequired)
+      .test('fileSize', 'File is too large.', checkFileSize),
+    status: yup.string().oneOf(['available', 'unavailable']),
```

Comment:
```suggestion
    const ItemStatuses = {
      Available: 'available',
      Unavailable: 'available',
    };
    status: yup.string().oneOf(Object.values(ItemStatuses)),
```
Create enum `ItemStatuses ` in `src/utils/constants`